.github/ISSUE_TEMPLATE/05-dev-days-project.yml

```diff
@@ -0,0 +1,74 @@
+name: Indicate your project is participating in Dev Days
+description: | 
+  For hosted projects only, use this form to indicate that your project will be participating in Dev Days.
+title: "Dev Days 2024 Project Application - PROJECTNAME"
+labels: ["help wanted"]
+assignees: 
+  - yarille
+  - jmertic
+  - carolalynn
+  - lgritz
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ![](https://artwork.aswf.io/other/dev-days/2024/color/dev-days-2024-color.svg)
+
+        Thank you for your interest in being a project wanting to participate in Dev Days 2024!
+        For hosted projects only, use this form to indicate that your project will be participating in Dev Days.
+
+        Please update `PROJECTNAME` in the title with your project name.
+  - type: input
+    attributes:
+      label: URL to your Dev Days 2024 landing page
+      description: |
+        This URL can be a webpage, wiki page, or file in your code repository, that outlines...
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: URL of issues for participants to work on
+      description: |
+        Projects should label issues with 'good first issue' and 'help wanted', and ensure 
+        that they are unassigned and no more than one year old. To validate, go to 
+        [CLOTributor](https://clotributor.dev/search?foundation=aswf&good_first_issue=true).
+
+        These issues should align with the Academy Software Foundation TAC's
+        [best practices for good first issues]()
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Communication channel(s) for partipants
+      description: |
+        What are the best communications channel(s) for Dev Days particpants to use? This is
+        likely your primary Slack channel or mailing list, but you could also setup a seperate
+        communication channel if that would be preferred.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Project meeting(s) for Dev Days partipants to join
+      description: |
+        What project meeting(s) should a potential Dev Days participant join if they want to 
+        work with your project during Dev Days? This might be one of the project's
+        regular TSC meetings, or could be separate meeting(s).
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: URL to the project "Getting Started" and build instructions
+      description: |
+        Indicate the URL where your project has instructions for Dev Days participants
+        to set up thier development environment and instructions for building the project.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected time commitments for participants
+      description: |
+        What should Dev Days partipants for the project expect in terms of time commitments? Note
```

Comment:
I'm not opposed to including this, but the time commitment section from my doc was for mentors/reviewers/maintainers from the project. Suggested wording:

"Please provide the name/email for you project's primary Dev Days contact (should be a TSC member/maintainer)"
Input: name / email

"In order to ensure a supportive environment for Dev Days participants, we suggest a minimum of three project contributors be available during Dev Days to help debug, review, and generally support participants. Names are not required at this time, but should be added to your Dev Days landing page before the event."

Inputs: acknowledge check box?

Reply:
I actually like the idea of removing the question; it makes more sense on a per-issue basis tbh.

In terms of data collecting, we can't ask for an email here. But the submitter will at least have their GH handle assigned to the issue, so you could communicate with them that way if that could work. LMK.

Reply:
One of the most important parts of this is knowing who the point person is that we can go to for anything we need communication wise.

Not sure it's worth it if we can't have that.

Reply:
We can ask for a name, just not an email. We do have the GH username, which will be attached to the submission.

Reply:
I guess we can assume the person filling out the form is the point person... and leave an option to put a different name.

I mean eventually we're going to need their email.... so if that's the case, is this the best home for this?

Reply:
The good news is you likely know all the folks submitting - so that's helpful.

I guess let's play it by ear and see what happens. Worst case we have to reach out to a few people.

Reply:
Okie dokie.